pebbles/static/js/controllers/DashboardController.js

```diff
@@ -192,6 +192,10 @@ app.controller('DashboardController', ['$q', '$scope', '$routeParams', '$interva
         $scope.provision = function (blueprint) {
             instances.post({blueprint: blueprint.id}).then(function (response) {
                 $scope.updateInstanceList();
+                $timeout(function () {
+                    instances.get(response.id, {'check_provisioning': true}).then(function (response) {
+                    });
+                }, 300000);
```

Comment:
Might increase the timeout to 10 mins, we might get a lot of mails with 5 mins :D

Reply:
Ok.